src/modules/Builder/features/Activities/ActivitiesHeader/ActivitiesHeader.utils.tsx

```diff
@@ -1,22 +1,42 @@
 import i18n from 'i18n';
 import { PerfTaskType } from 'shared/consts';
+import { useFeatureFlags } from 'shared/hooks/useFeatureFlags';
 
 import { ActivityAddProps, PerformanceTasks } from '../Activities.types';
 
 const { t } = i18n;
 
 const getMenuItemTitle = (perfTask: string) => `performanceTasks.${perfTask}`;
 
-export const getPerformanceTasksMenu = (
+export const GetPerformanceTasksMenu = (
```

Comment:
I understand why you renamed this, but this is neither a Component nor a Hook, violating [rules-of-hooks](https://react.dev/reference/rules/rules-of-hooks). You could instead pass the condition as a parameter.